platform/viewer/src/appExtensions/MeasurementsPanel/index.js

```diff
@@ -7,10 +8,34 @@ export default {
    */
   id: 'measurements-table',
 
-  preRegistration({ servicesManager, configuration = {} }) {
-    init({ servicesManager, configuration });
+  preRegistration({ servicesManager, commandsManager, configuration = {} }) {
```

Comment:
An aside -- We should maybe also pass in `rootConfig` or `applicationConfig`

Reply:
Added in the comment. 👍